src/util/00-mesa-defaults.conf

```diff
@@ -636,7 +643,9 @@ TODO: document the other workarounds.
             <option name="radeonsi_zerovram" value="true" />
         </application>
         <application name="Counter-Strike Global Offensive" executable="csgo_linux64">
+            <option name="mesa_glthread" value="true" />
```

Comment:
I was pinged over the weekend by a user in a CS:GO issue. It seems there's some concern about the two new options here (`mesa_glthread` and `radeonsi_clamp_div_by_zero`) affecting VAC scores: https://github.com/ValveSoftware/csgo-osx-linux/issues/2630